# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/build/nsLayoutStatics.cpp
# Commit: e21ad27bfd0a
# Full Hash: e21ad27bfd0a2fef90919101eaef5aa5af1cc6c2
# Author: Christoph Kerschbaumer <ckerschb@christophkerschbaumer.com>
# Date: 2019-10-22 21:43:14
# Regressor Bug: 1584993
# File Overlap Count: 1
# Description:
#   Bug 1584993: Make CSP frame-ancestors work with fission enabled. r=jkt,farre,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D49147
# ==============================================================================

diff -r d9d678e7422e -r e21ad27bfd0a layout/build/nsLayoutStatics.cpp
--- a/layout/build/nsLayoutStatics.cpp	Tue Oct 22 09:37:38 2019 +0000
+++ b/layout/build/nsLayoutStatics.cpp	Tue Oct 22 10:57:43 2019 +0000
@@ -109,6 +109,7 @@
 #include "DecoderDoctorLogger.h"
 #include "MediaDecoder.h"
 #include "mozilla/ClearSiteData.h"
+#include "mozilla/dom/DOMSecurityManager.h"
 #include "mozilla/EditorController.h"
 #include "mozilla/Fuzzyfox.h"
 #include "mozilla/HTMLEditorController.h"
@@ -308,6 +309,8 @@
 
   ClearSiteData::Initialize();
 
+  DOMSecurityManager::Initialize();
+
   // Reporting API.
   ReportingHeader::Initialize();
 